src/js/populateBoard.js

```diff
@@ -0,0 +1,30 @@
+import { getLeaderBoard } from './API';
+
+function createElementBoard(leaderboardHTMLTag, scores) {
+  leaderboardHTMLTag.innerHTML = '';
+  scores.sort((a, b) => parseInt(b.score, 10) - parseInt(a.score, 10));
+  scores.forEach((item) => {
+    const leaderboardItem = document.createElement('tr');
+
+    const nameContainer = document.createElement('td');
+    nameContainer.classList.add('item-player');
+    nameContainer.innerHTML = item.user;
+
+    const scoreContainer = document.createElement('td');
+    scoreContainer.classList.add('item-score');
+    scoreContainer.innerHTML = item.score;
+
+    leaderboardItem.appendChild(nameContainer);
+    leaderboardItem.appendChild(scoreContainer);
+
+    leaderboardHTMLTag.appendChild(leaderboardItem);
+  });
+}
+
+function populateBoard(leaderboardHTMLTag) {
```

Comment:
- [ ] Please make use of the arrow function instead of the function keyword. For more information on how the ES6 arrow function syntax works, see this [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

Reply:
Thanks for your feedback @Ghiftee, It is now solved on commit f0d37df.